### ISSUE TITLE
Fronted partitives

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/cgel#78]
- Out of Carnival Royal Caribbean and Norwegian cruises, which is the best and why?

The PP acts as a partitive: *which (out) of A and B cruises*

p. 903 discusses *which* + partitive, but not with the partitive fronted.

We agreed to call it a Supplement for now. Seems like too much of a stretch to say there is a gap, considering that fronted partitives seem to have a pragmatic rather than syntactic relationship to their member-referring expression:

- Of John, Mary, and Sue, I have always felt that Mary is the smartest.
- I have always felt that of John, Mary, and Sue, Mary is the smartest.
- Of her friends, I would say that none are qualified for the position.

**Comment by BrettRey** [comment]
The fact that the object of _of_ has to be definite or a coordination defining the set by listing its members is a strike against the supplement analysis, though this too could simply be pragmatics.
Geoff says, "I think it has to be either PP preposing or PP right-shifting, and in either case I think there has to be a gap."

**Comment by nschneid** [comment]
"Of John, Mary, and Sue, I have always felt that Mary is the smartest."—where would the gap be?

**Comment by nschneid** [comment]
Pragmatically it feels related to topicalization.

**Comment by BrettRey** [comment]
Granted, no gap there.

**Comment by nschneid** [comment]
It seems related to a topic-setting preferatory phrase: 
- When it comes to the three candidates, John says he thinks we should hire Mary.

There's a meaning relationship in that "three candidates" sets up an expectation that we may drill down into the set (in this case, focusing on Mary). But it doesn't seem like a *syntactic* gap. There is no guarantee that any members of the set will be mentioned:
- When it comes to the three candidates, I was pretty disappointed.

"Of" creates a stronger expectation of discussing or at least implying members of the set:
- #Of the three candidates, I was not very impressed.
- #Of the three candidates, the right decision is not obvious.
- Of the three candidates, I am not sure (who should be chosen).
- Of the three candidates, the idea that Mary is most qualified is absurd.

Note that in this last one, Mary is the subject of a nominal complement clause. This would be an island to WH-movement:

- *Who is the idea that __ is most qualified is absurd?

I think the pragmatics of the fronted partitive construction require that a member of the set receive focus later in the sentence. But I'm not convinced it's syntactically governed in the way that gaps are.

**Comment by nschneid** [comment]
Yep, those are the "standard" partitive construction, which to me is distinct from the fronted one.

@aryamanarora pointed out that the fronted one is similar to things like

- Between John and Mary, I have a slight preference for Mary.
- Between the two candidates, I have a slight preference for Mary.

The partitive meaning is here as well, only with "between" rather than "of".

**Comment by BrettRey** [comment]
If one is an adjunct and one a complement, then it seems we should be able to have both at once, but I don't think we can. 

Aryaman's examples, I can't get ?_I have a slight preference Between John and Mary for Mary._ So I don't find it relevant.

**Comment by nschneid** [comment]
I'm arguing that the sentence-initial ones are a construction where a sentence can begin by discussing a set, whose members will be commented on later. Agreed that it's odd to move the *between*-PP later in the sentence. But I think that's true for some of the sentence-initial *of* examples:

- Of John and Mary, I have a slight preference for Mary.  
   *I have a slight preference of John and Mary for Mary.
   *I have a slight preference for Mary of John and Mary. (could be grammatical but under a different reading, clarifying which Mary we're talking about)

If we can't have both sets at once, that could be due to the meaning. That said, this doesn't sound terrible (just a bit redundant):

- Of the three candidates, I am not sure who I prefer of the group.

And they could certainly be combined if one set is a superset of the other:

- Of the ten pieces, I am not sure which I prefer of the Monets (I have a clear favorite among the Rembrandts).

**Comment by BrettRey** [comment]
I think that the complement is licensed by a fused head specifically, and that anywhere there is a fused head with a fronted partitive, there's a gap. But there certainly do seem to be other cases where a fronted partitive must be an adjunct. 

I agree that _Of the ten pieces, I am not sure which I prefer of the Monets_ is not very bad.

**Comment by nschneid** [comment]
What do you think of:

- Of all the candidates with degrees in chemical engineering, which of them is your favorite?

**Comment by BrettRey** [comment]
Feels like a resumptive gap fill, like Paul McCartney's "world in which we live in".

**Comment by nschneid** [comment]
I can't decide if it's resumptive or just a bit redundant (on account of having two prepositions referring to the same partitive relationship).
